raiden-ts/src/channels/epics.ts

```diff
@@ -767,14 +767,15 @@ export const channelCloseEpic = (
         return of(channelClose.failure(error, action.meta));
       }
 
+      const balanceProof = channel.partner.balanceProof;
```

Comment:
My reason for this comment mostly is that we have unnecessary repetition, plus it's easier for the eye to focus without the repetition. thank you.